Sandbox/TicTacToe/TicTacToe.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+
+namespace TicTacToe
+{
+    class Program
+    {
+        public static string x = "x";
+        public static string y = "y";
+        public static int row = 0;
+        public static int column = 0;
+
+        public static bool gameWon = false;
+
+        public static string playerTurn = "x";
+        public static string[][] board = new string[][]
+        {
+            new string[] {" ", " ", " "},
+            new string[] {" ", " ", " "},
+            new string[] {" ", " ", " "}
+        };
+
+        public static void Main()
+        {
+            do
+            {
+                DrawBoard();
+                GetInput();
+                placeMarkForPlayer();
+                
+
+            } while (!CheckForWin() && !CheckForTie());
+
+            // leave this command at the end so your program does not close automatically
+            Console.ReadLine();
+        }
+
+        public static void GetInput()
+        {
+            
+            Console.WriteLine("Player " + playerTurn);
+            Console.WriteLine("Enter Row:");
+            row = int.Parse(Console.ReadLine());
+            Console.WriteLine("Enter Column:");
+            column = int.Parse(Console.ReadLine());
+            
+
+
+        }
+
+        public static void placeMarkForPlayer()
+        {
+            
+            PlaceMark();
+            setTogglePlayerTurn();
+
+
+        }
+
+
+        public static void setTogglePlayerTurn()
+        {
+           playerTurn = (playerTurn == x) ? y : x;
+
+        }
+
+      
+        public static void PlaceMark()
+        {   
+            board[row][column] = playerTurn;
+        }
+
+        public static bool CheckForWin()
+        {
+            
+            if(HorizontalWin() || VerticalWin () || DiagonalWin() )
+            {
+                gameWon = true;
+                return gameWon;
+               // Console.WriteLine("Player {0}", playerTurn + " Won!!! ");
+            }
+
+            else
+            {
+             return gameWon;  
+            }
+
+            
+        }
+
+        public static bool CheckForTie()
```

Comment:
Nice job with methods that explain what is going on so you don't have to write comments everywhere!